licenses/BSD-3-Clause

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
I believe this is same with BSD license which already in official portage tree, so drop it